lib/love-you.json

```diff
@@ -40,7 +40,7 @@
   "chichewa": "Ndimakukonda",
   "chickasaw": "Chiholloli",
   "chinese traditional": "我愛你",
-  "chinese simplified": "我愛你",
+  "chinese simplified": "我爱你",
```

Comment:
@John-Lin Check out [this diff](https://github.com/IonicaBizau/love-you/pull/2/files#diff-a23d65f62a73955e1526ecbd7391c307R43) from #2. Not quite sure, because I don't know Chinese... :thought_balloon: